docs/radio.rst

```diff
@@ -0,0 +1,67 @@
+Radio
+*****
+
+.. py:module:: radio
+
+The ``radio`` module allows devices to work together via simple wireless
+networks.
+
+Functions
+=========
+
+.. py:function:: send(message)
+
+    Sends a message, expressed as a string.
+
+.. py:function:: receive()
+
+    Receive the next incoming message on the message queue. Returns ``None`` if
+    there are no pending messages.
```

Comment:
@dpgeorge I agree that we should be explicit - an oversight on my part. 

Folks, I'm going to play devil's advocate here: if I were an 11yo the concept of a string of characters is understandable but the abstraction of such strings encoded as bytes is perhaps a level of abstraction too far *at this stage* (it's definitely something they should know about). Also, as a programmer I might want to both send and receive bytes.

I'm trying to find a way to allow both the naive 11yo sending/receiving of a message (as a student, I can `send("a string")` so it's probably reasonable to expect to receive messages as strings) and the correct / more powerful sending and receiving of bytes.

I wonder if we could wrap byte related functions with string related ones as a convenience for kids? For example:

```
$ radio.send("Hello, World")
$ radio.receive()
'Hello, back!'
$ radio.send(b'bytes')
$ radio.recv_bytes()
b'ack'
```

Note the *obvious* (to a naive user) names for sending and receiving strings. The `send` method is overloaded to automatically decode the string to bytes if a string is passed in. The `recv_bytes` method returns bytes and, since this is for "advanced" users, we signal it as such by truncating to "recv" and mentioning "bytes". under the hood, the `receive` method calls `recv_bytes` and attempts to encode the result as a string (I guess raising a `UnicodeEncodeError` if it fails).

Thoughts..?